src/main/java/mate/controller/DoneActionController.java

```diff
@@ -0,0 +1,18 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = "/done")
+public class DoneActionController extends HttpServlet {
```

Comment:
I think this class is redundant. Let's better redirect to the previous page when we do some actions.
E.g. after creating cars will redirect to all cars page, after creating new driver - redirects to the all drivers page etc.